packages/create-contentful-app/README.md

```diff
@@ -0,0 +1,8 @@
+# Contentful App Create React App template
```

Comment:
This readme is rather bare-bone. At the same time do we really need to mention CRA here?